packages/ui/src/theme/tokens/components/link.ts

```diff
@@ -9,13 +9,48 @@ interface LinkSizeTokens {
 }
 
 export interface LinkTokens {
-  active: LinkStateTokens;
+  /**
+   * @deprecated to be removed in the next major version please use _active
+   */
+  active?: LinkStateTokens;
   color: DesignToken<ColorValue>;
-  focus: LinkStateTokens;
-  hover: LinkStateTokens;
-  visited: LinkStateTokens;
+  /**
+   * @deprecated to be removed in the next major version please use _focus
+   */
+  focus?: LinkStateTokens;
+  /**
+   * @deprecated to be removed in the next major version please use _hover
+   */
+  hover?: LinkStateTokens;
+  /**
+   * @deprecated to be removed in the next major version please use _visited
+   */
+  visited?: LinkStateTokens;
+  _active?: LinkStateTokens;
+  _focus?: LinkStateTokens;
+  _hover?: LinkStateTokens;
+  _visited?: LinkStateTokens;
 }
 
+export const DEPRECATED_LINK_TOKENS = [
```

Comment:
Can we type this as `WebDesignToken[]` so we can ensure the shape of the deprecated tokens follows the standard tokens?

Reply:
I've added a type for these, but since they aren't `WebDesignTokens` I created a new type for them.

Reply:
Also this has been renamed to more clearly indicate that these are duplicate states and not just general purpose deprecated tokens